x-pack/plugins/actions/server/action_type_registry.ts

```diff
@@ -9,7 +9,6 @@ import Boom from '@hapi/boom';
 import { i18n } from '@kbn/i18n';
 import { RunContext, TaskManagerSetupContract } from '@kbn/task-manager-plugin/server';
 import { LicensingPluginSetup } from '@kbn/licensing-plugin/server';
-import { rawConnectorSchema } from './raw_connector_schema';
```

Comment:
Removing `rawConnectorSchema`  makes `raw_connector_schema` file redundant but i am leaving it there as we will use it for modelVersions.